src/main/java/com/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.wolox.training.repositories;
+
+import com.wolox.training.model.Book;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+public interface BookRepository extends CrudRepository<Book, Long> {
+
+    Book findByAuthor(@Param("author") String author);
```

Comment:
I recommend the use of Optional to avoid future null checks `Optional<Book>`, you can read more here https://www.baeldung.com/java-optional